backend/logic/users/users.js

```diff
@@ -1,94 +1,115 @@
-const mysql = require('../../sql/connection');
```

Comment:
Possible merge conflicts with #28. Will need to be resolved

Reply:
is it because of the single quotes? 0-0

Reply:
users.js is formatted in #28 and this

Reply:
we can merge this first, then I'll fix my commit #28, then we merge #28